addons/website/static/src/js/backend/button.js

```diff
@@ -40,5 +41,63 @@ var WidgetWebsiteButton = AbstractField.extend({
     },
 });
 
+var WidgetWebsiteButtonIcon = AbstractField.extend({
+    template: 'WidgetWebsiteButtonIcon',
+    fieldDependencies: _.extend({}, AbstractFieldBinary.prototype.fieldDependencies, {
+        website_url: {type: 'string'},
+    }),
+    events: {
+        'click': '_onClick',
+    },
+
+    /**
+    * Register some useful references for later use throughout the widget.
+    *
+    * @override
+    */
+   start: function () {
+       this.$icon = this.$('.o_button_icon');
+       return this._super();
+   },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    isSet: function () {
+        return true;
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+     /**
+     * @override
+     * @private
+     */
+    _render: function () {
+        this._super.apply(this, arguments);
+
+        var published = (this.value === true);
+        var info = published ? _t("Published") : _t("Unpublished");
+        this.$icon.prop('aria-label', info)
+                .prop('title', info)
```

Comment:
`aria-label` and `title` are properties?

Reply:
title is a property but not aria-label my bad.